bamboo/integration-tests.sh

```diff
@@ -4,5 +4,23 @@ set -ex
 . ./bamboo/set-bamboo-env-variables.sh
 . ./bamboo/abort-if-skip-integration-tests.sh
 
+if [[ $USE_CACHED_BOOTSTRAP == true ]]; then ## Change into cached cumulus dir
+  echo "*** Using cached bootstrap build dir"
+  cd /cumulus/
+fi
+
+### confirmLock will fail if another stack has lock, *redeploy* if no lock, and continue if a lock is in place already
```

Comment:
Do you mean it will fail if another job has locked the deployment already?

Reply:
Yes, good catch.